### PR TITLE
[NEW] Image description

### LINKED ISSUE #1166: [NEW] Add LaunchScreen

### PR DESCRIPTION
**Pull request by RahulDhiman93**
@RocketChat/ios

Closes #1166 
@cardoso

### CODE REVIEW

**Comment by cardoso** [comment]
Hi @RahulDhiman93 . Thanks for you contribution 😄 

Can you keep only the relevant changes for this PR?

reset the commits:
> git reset HEAD~3

add the relevant files:
> git add Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift
> git add Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.xib

reset the unneeded changes:
> git checkout .

commit and push:
> git commit -m "ChatMessageImageView Description"
> git push -f

**Comment by rafaelks** [comment]
@RahulDhiman93 Why do you have two PRs closing the same issue?

File: Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift [hunk 1]
```diff
@@ -60,5 +61,7 @@ final class ChatMessageImageView: UIView {
 
     @objc func didTapView() {
         delegate?.openImageFromCell(attachment: attachment, thumbnail: imageView)
+        labelTitle.isHidden = true
+        detail.isHidden = true
```
Comment (rafaelks) [inline Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift]: Why are you hiding it on tap? When do you show it again?

Comment (soundstorm) [reply Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift]: @rafaelks My suggestion was to *toggle* hide on tap on fullscreen as title and description [if displayed at all] would overlay the image. So maybe a misunderstanding :)

Comment (rafaelks) [reply Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift]: @soundstorm Gotcha, but only when in fullscreen, right? In this case, the labels are on the cell.

Comment (soundstorm) [reply Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift]: Yep, so a bit mixed up here :)

**Comment by rafaelks** [comment]
Closing it, due to inactivity.

**Pull request closed without merging**